doc/source/whatsnew/v1.4.0.rst

```diff
@@ -835,6 +835,7 @@ MultiIndex
 - Bug in :meth:`MultiIndex.get_loc` raising ``TypeError`` instead of ``KeyError`` on nested tuple (:issue:`42440`)
 - Bug in :meth:`MultiIndex.union` setting wrong ``sortorder`` causing errors in subsequent indexing operations with slices (:issue:`44752`)
 - Bug in :meth:`MultiIndex.putmask` where the other value was also a :class:`MultiIndex` (:issue:`43212`)
+- Bug in :meth:`MultiIndex.dtypes` when duplicate level names returned only one dtype (:issue:`45174`)
```

Comment:
Could you clarify a bit? Maybe only one entry for all levels with same name. I though this would return a Series with the correct length but with the same dtype in every row

Reply:
Hopefully done.